style.css

```diff
@@ -1 +1,270 @@
 /*  Add your styling here */
+/* styling header section */
+body{
+    margin:0px;
+    font-family: 'Langar', cursive;
+}
+header{
+    width:100%;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    align-items: flex-start;  
+  
+}
+.icon{
+    width:100%;
+    display: flex;
+    justify-content: space-between;
+    align-items: flex-start;
+    flex-wrap: nowrap;
+    margin:0px;
+}  
+.icon-img{
+  width: 15%;
+} 
+.icon-text{
+    width:90%;
+    text-align: center;
+ }
+.icon-img img{
+    width:100%;
+}
+.icon p{
+    font-size: calc(10px + 1vw);
+    width:100%;
+    color:rgb(143, 63, 76);
+}
+
+nav{
+    display: flex;
+    align-items: center;
+}
+
+#myLinks{
+    display:none;
+}
+
+.hamburger-icon{
+    margin-top:16px;
+    width: 60%;
+    align-items: flex-start;
+ }
+ .mobile-screen img{
+    background-color:pink;
+ }
+.topnav {
+    overflow: hidden;
+    background-color:pink;
+    position: absolute;
+    display:none;
+    min-width: 100vw;
+    right:0px;
+    top:calc(65px + 3vw);
+ }
+  
+
+ .topnav li{
+      list-style-type: none;
+ }
+ .topnav a {
+    color: black;
+    padding: 14px 16px;
+    text-decoration: none;
+    font-size: 17px;
+    display: block;
+ }
+ /* styling main section */
+.welcome-image{
+    max-width:100%;
+}
+.welcome-image:hover{
+    border:5px solid pink;
+}
+.welcome{
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    text-align: center;
+}
+.welcome h1{
+    font-size: calc(15px + 1.5vw);
+    color:rgb(143, 63, 76);
+    letter-spacing: 1vw;
+}
+.welcome-text p{
+    text-align: justify;
+    font-size: calc(10px + 1vw);
+    margin: 0px 10px;
+}
+.monthly-deals{
+    display: flex;
+    flex-direction: column;
+    display:none;
+    justify-content: center;
+}
+.monthly-deals p{
+    color:rgb(143, 63, 76);
+    font-size: calc(8px + 1vw);
+}
+.monthly-deals img:hover{
+    border:5px solid pink;
+} 
+.welcome-deal-wrapper{
+    width: 100%;
+    display:flex;
+    justify-content:space-around;
+    align-items: center;
+    flex-wrap:nowrap;
+}
+.products{
+    display: flex;
+    flex-direction: column;
+    row-gap:25px;
+}
+.square-cakes, .rounded-cakes{
+    display:flex;
+    flex-direction: column;
+    align-items: center;
+}
+
+
+.products img{
+    max-width:45%;
+} 
+
+.products img:hover{
+    border:5px solid pink;
+}
+.square-cakes p{
+    text-align:center;
+    font-size: calc(10px + 1vw);
+    color:rgb(143, 63, 76);
+}
+.rounded-cakes p{
```

Comment:
Check previous comment around duplicate styles

Reply:
I have done it thank you